blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/IntegrationTestBase.java

```diff
@@ -129,6 +129,26 @@ public void tearDown() throws Exception {
         // meh
     }
 
+    /**
+     * create a number of test locators of numTenants x numMetricNames combination
+     * @param numTenants
+     * @param numMetricNames
+     * @return list of locators for testing
+     */
+    protected List<Locator> generateTestLocators(int numTenants, int numMetricNames) {
```

Comment:
I'm thinking/debating/contemplating this method is specific to LocatorIOIntegrationTest. It's not very likely that other tests want to generate locators like this. Maybe we should move it back to LocatorIOIntegrationTest.